synth.py

```diff
@@ -83,6 +94,21 @@ def lowpass_noise(cutoff, duration, samplerate=SAMPLERATE):
     return noise
 
 
+@lru_cache()
+def bandpass_noise(cutoffl, cutoffh, duration, samplerate=SAMPLERATE):
+    frames = int(duration*samplerate)
+    print('generating some noise frames', frames)
+    noise = np.random.normal(0, 0.2, frames)
+    print('fft...')
+    fd_noise = np.fft.rfft(noise)
+    freq = np.fft.rfftfreq(noise.size, d=1/samplerate)
+    fd_noise[freq < cutoffl] = 0
+    fd_noise[freq > cutoffh] = 0
+    noise = np.fft.irfft(fd_noise)
+    print('got some noise')
```

Comment:
I would remove the prints.

Reply:
Indeed. I've left them just because they were already there